client/client_test.go

```diff
@@ -4672,6 +4673,44 @@ func testBasicLocalCacheImportExport(t *testing.T, sb integration.Sandbox) {
 	testBasicCacheImportExport(t, sb, []CacheOptionsEntry{im}, []CacheOptionsEntry{ex})
 }
 
+func testBasicS3CacheImportExport(t *testing.T, sb integration.Sandbox) {
```

Comment:
This is only one case. @crazy-max do you want help for others?

Reply:
I think the basic one is fine for this use case: https://github.com/moby/buildkit/blob/05c4d571cf97fd0878fecc074972a410af889ca9/client/client_test.go#L4556-L4616

If you think about ones specific to s3, we can do that in follow-ups.